tests/test_replay.py

```diff
@@ -53,10 +66,10 @@ def test_is_perfect_combo(self):
 
     def test_nomod(self):
         for replay in self._replays:
-            self.assertEqual(replay.mods, Mod.NoMod, "Mod combination is wrong")
+            self.assertEqual(replay.mods.value, Mod.NoMod.value, "Mod combination is wrong")
```

Comment:
I assume this means the rust version doesn't support comparing intflag enums? That's pretty bad as well. I wouldn't want to use `.value` every time I did a mod comparison.